src/js/options.ts

```diff
@@ -147,10 +148,28 @@ function createPageWithPrefs(prefs:Options) {
     create(layout, 'range', { id: 'space', max: '200', min: '0', step: '5' }, chrome.i18n.getMessage('space'));
     create(layout, 'range', { id: 'fontsize', max: '150', min: '50', step: '10' }, chrome.i18n.getMessage('fontsize'));
     create(layout, 'checkbox', { id: 'useCustomScrollbar' }, chrome.i18n.getMessage('useCustomScrollbar'));
+    if (linksStats) {
+      const stats = create(settings, 'section', {}, chrome.i18n.getMessage('clicks_analytics'));
+      create(stats, 'table', {}, '');
+      populateStats(JSON.parse(linksStats));
+    }
   }
   updatePageWithPrefs(prefs);
 }
 
+function populateStats(linkStats: any) {
+  const tbody = document.getElementsByTagName('tbody')[0];
+  let html = '';
+  for (const [url, number] of Object.entries(linkStats).sort(function (a, b) {
+    const aVal = Object.values(a)[1] as number;
+    const bVal = Object.values(b)[1] as number;
+    return bVal - aVal;
+  })) {
+    html += '<tr><td>' + url + '</td><td>' + String(number) + '</td></tr>';
+  }
+  tbody.innerHTML = html;
```

Comment:
This panel is not a setting (i.e. it's not something for the user to tweak).  I suggest we remove it for now and add again later if there is a clear need.